### PR TITLE
packages for centos 6 & 7, push to debian/jessie too

### PR DESCRIPTION
**Pull request by ctdk**

### CODE REVIEW

File: scripts/package.sh [hunk 1]
```diff
@@ -44,4 +43,41 @@ fpm -s dir -t deb \
...
+mkdir -p ${BUILD}/usr/sbin
+mkdir -p ${BUILD}/etc/init
+mkdir -p ${BUILD}/etc/raintank
+
+cp ${BASE}/config/metrictank.ini ${BUILD}/etc/raintank/
+cp ${BUILD_ROOT}/metrictank ${BUILD}/usr/sbin/
+cp ${BASE}/config/upstart-0.6.5/metrictank.conf $BUILD/etc/init
+
+PACKAGE_NAME="${BUILD}/metrictank-${VERSION}.el6.${ARCH}.rpm"
+## CHECK IF THIS MAKES A WORKING RPM
```
Comment (Dieterbe) [inline scripts/package.sh]: this comment doesn't make sense. we still need to check if it works or not?

**Comment by Dieterbe** [comment]
so executing the build.sh now doesn't work anymore as the circleci check shows, could this be due to your change?

**Comment by woodsaj** [comment]
@Dieterbe the broken build was due to the move to vendoring.  I fixed that today, so if you pull in master this branch should build correctly.

**Comment by ctdk** [comment]
I was about to say that my changes to build.sh were minimal (I think it was limited to adding CGO_ENABLED=0).

**Comment by ctdk** [comment]
merged master, pushed this branch back up.

File: scripts/package.sh (hunk 1 above)
Comment (ctdk) [inline scripts/package.sh]: No, I just forget to remove the note to myself there.

**Comment by Dieterbe** [comment]
thanks!

**Pull request merged**